themes/fast-track-theme/lms/templates/ccx/team_management.html

```diff
@@ -132,11 +133,27 @@ <h2 class="hd hd-2">${_("Team management")}</h2>
                     var role = 'Facilitator'
             }
 
-            var newRow = '<tr><td>' + role + '</td><td>' + username + '</td><td>' + email + '</td><td><span class="remove-coach-button" data-remove-url="' + removeUrl + '" data-unique-student-identifier="' + username + '" data-rolename="ccx_coach" data-action="revoke"><i class="fa fa-times-circle remove-icon"></i>Remove</span></td></tr>';
+            debugger;
+
+            if(isStaff) {
+               var newRow = '<tr><td>' + role +
+                '</td><td>' + username +
+                '</td><td>' + email +
+                '</td><td><a href="/affiliates/edit/' + username + '"><i class="fa fa-edit"></i>Edit</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;' +
```

Comment:
Basically, show Edit button for staff user since they see it in pre-rendered entries.